File: week-1/rotate_array_one/rotate_array_one.c
#include<stdio.h>

int main() {

    int n;

    printf("Enter the number of items in the array : ");
    scanf("%d",&n);


    int arr[n];
    printf("Enter the array elements : ");
    for(int i=0; i<n; i++) {
        scanf("%d",&arr[i]);
    }

    int last = arr[n-1];


    for(int i=n-1; i>0; i--) {
        arr[i] = arr[i-1];
    }

    arr[0] = last;

    printf("Array after rotation : ");
    for(int i=0; i<n; i++) {
        printf("%d ",arr[i]);
    }

    return 0;
}

File: week-1/rotate_array_one/rotate_array_one.py
n = int(input("Enter the size of the array : "))

arr = list(map(int, input("Enter the elements : ").split()))

last = arr[-1]

for i in range(len(arr)-1, 0 , -1):
    arr[i] = arr[i-1]

arr[0] = last

print("Array after rotation : ",arr)